onthego/youtube.py

```diff
@@ -72,7 +72,10 @@ def download_to_tmp(self, track):
 
         tmp_path = get_tmp_path(best)
         print("    Downloading %s to %s" % (video_url, tmp_path))
-        best.download(tmp_path, quiet=True)
+        try:
+            best.download(tmp_path, quiet=True)
+        except IOError:
+            return None
```

Comment:
This will trigger the printing of a "---- No You Tube video found" message in the `Downloader.audio` method. I think it's better to let the `download_to_tmp` method raise an error and catch this error in the `audio` method. But what behaviour should be adopt after the exception was caught? Should we try to:

1. download the file again?
2. quit the program?
3. keep running as if nothing happened?

I don't have a definite answer, but 3. does not seem right to me.